src/app-layer-ftp.c

```diff
@@ -536,7 +537,7 @@ static uint32_t CopyCommandLine(uint8_t **dest, const uint8_t *src, uint32_t len
         *dest = where;
     }
     /* either 0 or actual */
-    return length;
+    return length ? length + 1 : length;
```

Comment:
Would `return length ? length + 1 : 0;` be the same? Seems more clear that 0 will be returned then.

Reply:
Yes, exactly the same.